R/commonMachineLearningRegression.R

```diff
@@ -114,9 +114,9 @@
     if (any(predictorsAreFactors)) {
       predictorFactorsWithUniqueLevels <- which(sapply(predictorData[, predictorsAreFactors, drop = FALSE], FUN = function(x) length(x) == length(unique(x))))
       if (length(predictorFactorsWithUniqueLevels) == 1) {
-        jaspBase:::.quitAnalysis(gettextf("The predictor %1$s is a factor with unique levels, please remove this factor as a predictor.", names(predictorFactorsWithUniqueLevels)))
+        jaspBase:::.quitAnalysis(gettextf("The features %1$s is a factor with unique levels, please remove this factor as a feature.", names(predictorFactorsWithUniqueLevels)))
```

Comment:
feature(?) 

What does unique levels refer to here? Do you mean that there is only one level in this factor?